internal/server/providers/oidc.go

```diff
@@ -110,7 +110,7 @@ func (o *oidcClientImplementation) Validate(ctx context.Context) error {
 				return newValidationError("clientSecret")
 			}
 		}
-		logging.Debugf("%s", err.Error())
+		logging.Tracef("validating oidc provider resulted in: %s. this may be expected.", err.Error())
```

Comment:
Not blocking , but it might be nice to limit the number of helpers we have in `logger.go`, and use the `zerolog` interface directly as much as possible. I think those helpers were mostly to migrate from the previous logger.

```suggestion
		logging.L.Trace().Err(err).Msg("validating oidc provider. An error may be expected.")
```

That allows users to find the error in a predictable place in the structured json output.

We should remove `Tracef` if we go with this approach.

Reply:
Good suggestion, made this change.